pkg/command/output.go

```diff
@@ -32,11 +34,33 @@ type Printer interface {
 	Fprint(out io.Writer, obj interface{}) error
 }
 
+// formatTime is a helper that returns empty strings for zero times and adds
+// support for a humanized format (if the layout is empty).
+func formatTime(t time.Time, layout string) string {
+	switch {
+	case t.IsZero():
+		return ""
+	case layout == "":
+		return humanize.Time(t)
+	default:
+		return t.Format(layout)
+	}
+}
+
+// NOTE: All the "*Row" structs have `json:"-"` for everything EXCEPT their
+// inline "*Item" field so when the row is marshalled as JSON it appears the
+// same as what the item would have been.
```

Comment:
This is a reminder for me too. I couldn't remember why I had a those `json:"-"` tags; but the `*Row` objects are passed straight to the JSON marshaller. The alternative would have been a custom JSON marshaller function that just marshalls the item field.